lib/diagnostic.rb

```diff
@@ -108,10 +109,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'Hash.new; person_hash = { favorite_number: 2, first_name: 'Angela'}'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'Hash.new('#{first_name} is nice')'
```

Comment:
Here again we wanted the short hand syntax not the constructor:

`'person.default = "#{person[:first_name]} is nice"'`